dll/win32/shell32/wine/pidl.h

```diff
@@ -285,6 +290,7 @@ LPITEMIDLIST	_ILCreateGuidFromStrW(LPCWSTR szGUID) DECLSPEC_HIDDEN;
 LPITEMIDLIST	_ILCreateDesktop	(void) DECLSPEC_HIDDEN;
 LPITEMIDLIST	_ILCreateFromFindDataW(const WIN32_FIND_DATAW *stffile) DECLSPEC_HIDDEN;
 HRESULT		_ILCreateFromPathW	(LPCWSTR szPath, LPITEMIDLIST* ppidl) DECLSPEC_HIDDEN;
+LPITEMIDLIST SHELL32_CreateSimpleIDListFromPath(LPCWSTR pszPath, DWORD dwAttributes) DECLSPEC_HIDDEN;
```

Comment:
I'm doubtful these DECLSPEC_HIDDEN serve any actual purpose.
Also, Wine cleans them up: https://github.com/search?q=repo%3Awine-mirror%2Fwine+DECLSPEC_HIDDEN&type=commits

Reply:
I assume it has something with GCC not exporting them but I don't know if it still matters, I was just following the existing style there.